lavalink/client.py

```diff
@@ -203,11 +229,18 @@ def add_node(self, host: str, port: int, password: str, region: str,
         :param event:
             The event to dispatch to the hooks.
         """
-        for hook in self._event_hooks:
-            try:
+        unknown_events = self._event_hooks.get(None) or []
+        registered_events = self._event_hooks.get(event) or []
+
+        event_hooks = unknown_events + registered_events
+
+        try:
+            for hook in event_hooks:
```

Comment:
Better was so we don't need to make two shallow copies of both lists and just use itertools:
`for item in itertools.chain(unknown_events, registered_events):`

Reply:
Must be two for loops or keep it like this if we are going to make the for loop async.

Reply:
Wdym? Iterator has nothing to do with async

Reply:
Linter picks up that `itertools.chain()` is not a `AsyncIterable`, or something like that. Thats why in my latest commits I just created two async for loops, other than going through the hassle of doing a bunch of these things.

Reply:
There's no need to have `async for` since lists you're putting in are not awaitable

Reply:
Oh okay, I'll fix it up tomorrow